content/hi/docs/concepts/architecture/control-plane-node-communication.md

```diff
@@ -0,0 +1,119 @@
+---
+reviewers:
+- dchen1107
+- liggitt
+title: नोड्स और कंट्रोल प्लेन के बीच संचार
+content_type: concept
+weight: 20
+aliases:
+- master-node-communication
+---
+
+<!-- overview -->
+
+यह दस्तावेज़ एपीआई सर्वर और कुबेरनेट्स क्लस्टर के बीच संचार पथों को सूचीबद्ध करता है।
+इरादा उपयोगकर्ताओं को नेटवर्क कॉन्फ़िगरेशन को सख्त करने के लिए उनकी स्थापना को अनुकूलित करने की अनुमति देना है
+जैसे कि क्लस्टर को एक अविश्वसनीय नेटवर्क (या क्लाउड पर पूरी तरह से सार्वजनिक आईपी) पर चलाया जा सकता है
+प्रदाता)।
+
+<!-- body -->
+
+## विमान को नियंत्रित करने के लिए नोड
```

Comment:
Consider incorporating English text for the link to avoid non-English characters in the URL of the section.

```suggestion
## विमान को नियंत्रित करने के लिए नोड {#node-to-control-plane}
```